### PR TITLE
Repair broken link for first contributions

### LINKED ISSUE #6493: The link to "Contributor Role Board" in the https://www.kubernetes.dev/docs/guide/first-contribution/ page is broken.
**This is a Bug Report**

**Problem:**
The link to "Contributor Role Board" in the https://www.kubernetes.dev/docs/guide/first-contribution/ page is broken.
**Proposed Solution:**
The page should redirect to correct public page
**Page to Update:**
https://kubernetes.io/...

### PR DESCRIPTION
**Pull request by uroy-personal**
The link to "Contributor Role Board" in the https://www.kubernetes.dev/docs/guide/first-contribution/ page is broken.  The PR updates it to the good first issue link.

Fixes #6493

### CODE REVIEW

File: contributors/guide/first-contribution.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ subrepository. For example, a documentation issue should be opened in
 [kubernetes/website]. Make sure to adhere to the prompted submission guidelines 
 while opening an issue. Check the [issue triage guide] for more information.
 
-[Contributor Role Board]: https://discuss.kubernetes.io/c/contributors/role-board
+[Community Issues]: https://github.com/kubernetes/community/issues
```
Comment (mrbobbytables) [inline contributors/guide/first-contribution.md:130]:
```suggestion
[issues labeled as a good first issue]: https://go.k8s.io/good-first-issue
```

corresponding link

File: contributors/guide/first-contribution.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ Here are some things you can do today to get started contributing:
 * Help triage issues
 
 If the above suggestions don't appeal to you, you can browse the 
-[Contributor Role Board] to see who is looking for help. Those interested 
+[Community Issues] to see who is looking for help. Those interested
```
Comment (mrbobbytables) [inline contributors/guide/first-contribution.md:33]:
```suggestion
[issues labeled as a good first issue] to see who is looking for help. Those interested 
```

We should direct folks to any good first issue 👍

**Review by mrbobbytables — CHANGES_REQUESTED** [review]
Thanks for the fix, one suggestion - we should direct folk to any good first issue or help wanted issue, and not directly to k/community. There are a lot of other places throughout the project that have them :)

**Comment by uroy-personal** [comment]
> Thanks for the fix, one suggestion - we should direct folk to any good first issue or help wanted issue, and not directly to k/community. There are a lot of other places throughout the project that have them :)

Thanks @mrbobbytables,
Appreciate your suggestions. Have committed the changes.

**Review by uroy-personal — COMMENTED** [review]
Suggestions accepted.

**Comment by tpepper** [comment]
This looks good to me, but we'll want to squash and merge and for that the PR should have a clean description which will be recorded with the merge commit.  Can you either manually squash into a single commit with a clean description, or edit the description of the PR in GitHub to include even just a copy/paste of the description from https://github.com/kubernetes/community/issues/6493 ?

**Comment by uroy-personal** [comment]
Hi @tpepper ,
Thanks for the suggestion.
The pull request description has been updated to make it same as the description of the issue covered by the pull request.

**Pull request merged**